pkg/boot/universalpayload/handoffblock.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package universalpayload
+
+import (
+	"strings"
+	"unsafe"
+
+	guid "github.com/google/uuid"
+	"github.com/u-root/u-root/pkg/boot/kexec"
+)
+
+// Efi HOB Handoff Info Table
+const EfiHobHandoffTableVersion = 0x0009
```

Comment:
Hob I thought meant Hand Off Block? So this is Hand Off Block Handoff? That seems a bit odd to me.

Reply:
Yes, you're right that HOB means Hand-Off Block according to spec: https://www.intel.com/content/dam/doc/reference-guide/efi-hand-off-block-specification.pdf 
As for the naming `EfiHobHandoffTableVersion`, we think the prefix `EfiHob` indicates the variables or structs used to define a HOB. We have many HOBs, and the `Hand-Off Table` is a special HOB expected to be in the first place. 
And also the [edk2 repo](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/Pi/PiHob.h#L54) keeps the same naming style. I think we should follow the same way.